micronota/__init__.py

```diff
@@ -0,0 +1,10 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2015--, micronota development team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file COPYING.txt, distributed with this software.
+# ----------------------------------------------------------------------------
+
+__credits__ = "https://github.com/biocore/micronota/graphs/contributors"
+__version__ = "0.1.0.dev0"
```

Comment:
`dev0` is actually one of standard way to version string: https://www.python.org/dev/peps/pep-0440/